### ISSUE TITLE
PHP8 error

### ISSUE THREAD

**Issue by grezniczek** [Unisante/Barcode-Scanner#2]
When the module is enabled, but not used, $em_fields will be null, causing an error in PHP8 further down in line 39.

https://github.com/Unisante/Barcode-Scanner/blob/86646a0d9805b59aa0f6b311507ff00778ee70dd/Scan.php#L23

It is probably sufficient to change this to:
```php
$em_fields = $this->getProjectSetting('scannable-field-list') ?? []; 
```

**Comment by unisante-user** [comment]
Thank you for bringing this issue to our attention!
The fix has been included in the new release, [v1.1.2](https://github.com/Unisante/Barcode-Scanner/releases/tag/1.1.2)
Additionally, the new release has been submitted to the REDCap Repo.
It should be available soon.

**Comment by grezniczek** [comment]
@Unisante - Thank you for the swift response!
